README.Rmd

```diff
@@ -40,15 +40,16 @@ If you are new to dplyr, the best place to start is the [data transformation cha
 
 In addition to data frames/tibbles, dplyr makes working with other computational backends accessible and efficient. Below is a list of alternative backends:
 
+- [arrow](https://arrow.apache.org/docs/r) for larger-than-memory datasets, including on remote cloud storage like AWS S3, using the Apache Arrow C++ engine, [Acero](https://arrow.apache.org/docs/cpp/streaming_execution.html).
+
 - [dtplyr](https://dtplyr.tidyverse.org/) for large, in-memory datasets. 
   Translates your dplyr code to high performance 
   [data.table](https://rdatatable.gitlab.io/data.table/) code.
 
 - [dbplyr](https://dbplyr.tidyverse.org/) for data stored in a relational 
   database. Translates your dplyr code to SQL.
 
-- [duckdb](https://duckdb.org/docs/api/r) or 
```

Comment:
I didn't realize arrow was already mentioned! Looks like Hadley added this just 2 weeks ago--maybe he was eavesdropping on our conversation!

It's odd (to me, in Hadley's change) that duckdb is listed separately from dbplyr since it uses dbplyr's SQL generation, but I guess the difference is that the duckdb package includes the query engine, you don't have to install a database separately.